db/seeds.rb

```diff
@@ -115,16 +115,24 @@
                              configuration_snapshot: 2, description: 'This is a test description.')
 
 # SearchFacet seeds
-SearchFacet.create(name: 'Species', identifier: 'species', filters: [{id: 'NCBITaxon_9606', name: 'Homo sapiens'}],
-                   ontology_urls: [{name: 'NCBI organismal classification', url: 'https://www.ebi.ac.uk/ols/api/ontologies/ncbitaxon'}],
-                   data_type: 'string', is_ontology_based: true, is_array_based: false, big_query_id_column: 'species',
-                   big_query_name_column: 'species__ontology_label', convention_name: 'alexandria_convention', convention_version: '1.1.3')
-SearchFacet.create(name: 'Disease', identifier: 'disease', filters: [{id: 'MONDO_0000001', name: 'disease or disorder'}],
-                   ontology_urls: [{name: 'Monarch Disease Ontology', url: 'https://www.ebi.ac.uk/ols/api/ontologies/mondo'},
-                                   {name: 'Phenotype And Trait Ontology', url: 'https://www.ebi.ac.uk/ols/ontologies/pato'}],
-                   data_type: 'string', is_ontology_based: true, is_array_based: true, big_query_id_column: 'disease',
-                   big_query_name_column: 'disease__ontology_label', convention_name: 'alexandria_convention', convention_version: '1.1.3')
-SearchFacet.create(name: 'Organism Age', identifier: 'organism_age', big_query_id_column: 'organism_age', big_query_name_column: 'organism_age',
-                   big_query_conversion_column: 'organism_age__seconds', is_ontology_based: false, data_type: 'number',
-                   is_array_based: false, convention_name: 'alexandria_convention', convention_version: '1.1.3', unit: 'years')
+species = SearchFacet.find_or_initialize_by(identifier: 'species')
+species.filters = [{id: 'NCBITaxon_9606', name: 'Homo sapiens'}]
+species.save
+
+disease = SearchFacet.find_or_initialize_by(identifier: 'disease')
+disease.filters = [{id: 'MONDO_0000001', name: 'disease or disorder'}]
+disease.save
+
+age = SearchFacet.find_or_initialize_by(identifier: 'organism_age')
+age.name = 'Organism Age'
+age.data_type = 'number'
+age.is_ontology_based = false
+age.is_array_based = false
+age.big_query_id_column = 'organism_age'
+age.big_query_name_column = 'organism_age'
+age.big_query_conversion_column ='organism_age__seconds'
+age.convention_name = 'Alexandria Metadata Convention'
```

Comment:
In [app/lib/search_facet_populator.rb line 76](https://github.com/broadinstitute/single_cell_portal_core/pull/429/files#diff-d2e81b6d4ff0bd3a2b5210c769a513a0R76), I see `facet.convention_name = 'Alexandria Convention'`.  So it seems either that should be "Alexandria Metadata Convention", or this should be "Alexandria Convention".

Reply:
Long story short - you are correct, but in `SearchFacetPopulator.populate_from_schema`, we use the `title` attribute from the convention JSON file.  That value is `Alexandria Metadata Convention`.  `SearchFacetPopulator.populate_sample_facets` is only referenced in a migration that gets clobbered completely by the next migration that runs.  So it really isn't used anywhere, and all the code that does get used has the value set to `Alexandria Metadata Convention`.

Having said that, I agree it's totally Department of Redundancy Department, and not even consistently so.  But if those values don't match, it causes some tests to fail, and the JSON is in source control in a different repository.

Reply:
And now that I've said all that, I agree that the inconsistency in the codebase (even in unused or deprecated methods) is overly confusing, so I've updated everything to be the same.